Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;net471;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net45;net471;netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Adding this explicitly to the test project is good for the extra test coverage, but I'm not sure it's necessary in the main project; I'd have expected it to already be 3.1 compatible based on the netstandard2.0 target. Is there something new in 3.1 motivating the specific target for netcore?

Reply:
Again correct :)